src/api/user/src/services/firestore.js

```diff
@@ -0,0 +1,30 @@
+const admin = require('firebase-admin');
+const { logger } = require('@senecacdot/satellite');
+
+// If a private key exists in env.development, run in online mode
+// else run in emulator mode
+if (process.env.PRIVATE_KEY) {
+  admin.initializeApp({
+    credential: admin.credential.cert({
+      type: process.env.TYPE,
```

Comment:
All of these env variables need to get added to the `config/env.staging` and `config/env.production` files with a line of documentation above them indicating what they mean, linking to docs, sample values, etc.

I've also filed https://github.com/Seneca-CDOT/telescope/issues/1932 to do the setup on staging/production.